environments/aks/preview.tfvars

```diff
@@ -42,7 +42,7 @@ clusters = {
     linux_node_pool = {
       vm_size   = "Standard_D8ds_v5"
       min_nodes = 30
```

Comment:
Should we update min? thinking just during a redeploy or such - adding time if we have like min of 50 or such